papas/mergedclusterbuilder.py

```diff
@@ -29,12 +29,12 @@ def __init__(self, clusters, ruler, history_nodes = None):
                 link_type = 'ecal_ecal', 'ecal_track' etc
                 is_link = true/false
                 distance = float
-        hist_nodes is an optional dictionary of Nodes : { id:Node1, id: Node2 etc}
+        history_nodes is an optional dictionary of Nodes : { id:Node1, id: Node2 etc}
             it could for example contain the simulation history nodes
             A Node contains the id of an item (cluster, track, particle etc)
             and says what it is linked to (its parents and children)
-            if hist_nodes is provided it will be added to with the new block information
-            If hist_nodes is not provided one will be created, it will contain nodes
+            if history_nodes is provided it will be added to with the new block information
```

Comment:
General comment: Could you please add periods at the end of sentences?
i have misread a few sentences in this PR because or this.